tests/integration/auth/auth.test.ts

```diff
@@ -508,7 +508,7 @@ describe('Auth routes', () => {
       expect(dbUser).toBeDefined()
       if (!dbUser) return
 
-      const isPasswordMatch = await bcrypt.compare(newPassword, dbUser.password)
+      const isPasswordMatch = await bcrypt.compare(newPassword, dbUser.password || '')
```

Comment:
Because the typing for the bcrypt compare function requires a `string` and the password is a `string | null` meaning if the password is `null` then it might not work properly so for that situation we replace it with an empty string.